ajax/libs/ng-browser-info/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "ng-browser-info",
+  "filename": "ngBrowserInfo.min.js",
+  "description": "AngularJS service that gives you a collection of methods for knowing more about your client browser ",
+  "license": "Apache-2.0",
+  "author": {
+    "name": "Uku Pattak",
+    "email": "uku.pattak@transferwise.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/transferwise/ng-browser-info"
+  },
+  "keywords": [
+    "client", "info", "useragent", "navigator", "angularjs", "angular", "os", "browser", "screen", "cookies"
```

Comment:
Let's make the one keyword in one line.
Such as https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/JQuery-Snowfall/package.json#L10-L14
Becuase it is useful to us to read.
Thank you.